SmartDeviceLink/SDLStreamingAudioLifecycleManager.m

```diff
@@ -155,6 +155,11 @@ - (void)didEnterStateAudioStreamStopped {
     _audioEncrypted = NO;
 
     [[NSNotificationCenter defaultCenter] postNotificationName:SDLAudioStreamDidStopNotification object:nil];
+
+    if (self.audioServiceEndedCompletionHandler != nil) {
+        self.audioServiceEndedCompletionHandler();
+        self.audioServiceEndedCompletionHandler = nil;
+    }
```

Comment:
Is this a part of this PR? If so, can you describe what this change is in the PR description / changelog?

Reply:
Added